### ISSUE TITLE
Include a search function to filter for specific tiles when in the look around menu.

### ISSUE THREAD

**Issue by DoctorBoomstick** [CleverRaven/Cataclysm-DDA#56507]
### Is your feature request related to a problem? Please describe.

I am a screen reader user who plays CDDA without the use of sight, and one of the biggest accessibility barriers for VI players that I’ve come across is unanimously finding tiles of importance, like lab stairs, computer consoles, soler panels, etc. Generally speaking, the regular way that a blind individual obtains info about the contents of a room they’re in is by using the look around menu, moving from tile to tile and examining each one of them to build a mental picture of the world around them. As you can imagine, this is very time consuming and tedious, as where as a sighted player can spend a few seconds poking around and getting info over where all of the important objects are, it’d take a blind player at least half an hour to receive the same depth of information. As such, because it’s impractical to look at every single tile within a standard over map location, numbering at least 625 individual tiles, vi players normally just check the tiles adjacent to walls if in a building, such as labs. Obviously, this leaves room for things to be missed and not seen by the player, and sometimes in situations like lab starts, it’s not uncommon for a VI player to walk by something like stairs or computer terminals, and simply not realise it’s there.

### Solution you would like.

I propose that a search functionality should be introduced into the look around menu, similar to what’s present on the over map. Essentially, the player would, after pressing x to bring up the look around panel, be able to press / like one is able to do on the over map, and filter for specific terrain or furniture. Like on the overmap, the player would be presented with a navigable menu of filtered results, which they could press enter on and be focused on the specific terrain tile. Basically, to put it in short my suggestion is to take the same exact search feature present on the over map, and introduce it to the look around menu. Such a change would make it far simpler to search for important land marks, and would make the game greatly more accessible.

### Describe alternatives you have considered.

Pinching a few million eye bionic implants from the exodii and distributing them for free around the blind community.

**Comment by mqrause** [comment]
I'm working on adding this to the `V` menu in #55503, it would work like checking nearby items. Progress is a bit slow currently because I want to improve UI code for this first.
